tools/update_tool

```diff
@@ -184,7 +198,7 @@ def filter_fn(imi):
                         if partition["fstype"] == "iso9660":
                             partition["uuid"] = "2022-07-01-fixed-uuid"
         return imi
-    return iso(lvm2(imi))
+    return iso(lvm2(tmpfiles_d(imi)))
```

Comment:
It would probably make sense to keep the "sanitization" code needed for diffing in a single place. You could for e.g. move it to a separate script that would be used for pre-processing the report before diffing. And then in any workflow, you would simply call it before diffing.